stateoftheuniverse/widgets/moon_functions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
```

Comment:
Is there a scenario when you want to directly run this file as a script in your terminal? If not, I don't think we need the Python shebang here 😉

Reply:
Habits die hard :)